pkg/reconciler/volumeclaim/pvchandler.go

```diff
@@ -62,10 +62,16 @@ func (c *defaultPVCHandler) CreatePersistentVolumeClaimsForWorkspaces(ctx contex
 		switch {
 		case apierrors.IsNotFound(err):
 			_, err := c.clientset.CoreV1().PersistentVolumeClaims(claim.Namespace).Create(ctx, claim, metav1.CreateOptions{})
-			if err != nil {
+			if err != nil && !apierrors.IsAlreadyExists(err) {
 				errs = append(errs, fmt.Errorf("failed to create PVC %s: %s", claim.Name, err))
 			}
-			if err == nil || !apierrors.IsAlreadyExists(err) {
+
+			if apierrors.IsAlreadyExists(err) {
+				c.logger.Infof("Tried to create PersistentVolumeClaim %s in namespace %s, but it already exists",
+					claim.Name, claim.Namespace)
+			}
```

Comment:
@cugykw could you please add a test case for this new check?

Reply:
@jerop ok.

Reply:
thank you @cugykw